mmcls/models/utils/data_preprocessor.py

```diff
@@ -135,12 +150,36 @@ def forward(self, data: dict, training: bool = False) -> dict:
             inputs = stack_batch(processed_inputs, self.pad_size_divisor,
                                  self.pad_value)
 
-        # ----- Batch Aug ----
-        if training and self.batch_augments is not None:
-            data_samples = data['data_samples']
-            inputs, data_samples = self.batch_augments(inputs, data_samples)
-            data['data_samples'] = data_samples
-
-        data['inputs'] = inputs
-
-        return data
+        data_samples = data.get('data_samples', None)
```

Comment:
To help the user for understanding our codebase, maybe we need to update the document by adding some description of the `data_samples` used in MMClassification(with text or link to MMEngine related document), this could be added into the document TODO list.

Reply:
We can create an issue and add it to Oct. TODO list